src/buildStyles.mjs

```diff
@@ -45,11 +45,13 @@ const buildStyles = async({
         const postCSSOptions = {
             from: file,
             to: destinationFilePath,
-            map: { prev: sassResult.sourceMap },
+            map: { prev: sassResult.sourceMap, inline: false, annotation: true },
         };
         const postCSSResult = await postcss([autoprefixer]).process(sassResult.css, postCSSOptions);
         postCSSResult.warnings().forEach((warning) => console.warn(warning.toString()));
 
+        sassResult.sourceMap.sources = [file];
```

Comment:
Hier überschreiben wir einen durch ein externes Tool generiertes Objekt. Gern noch kurz dokumentieren, wieso genau wir das machen, sonst wissen wir's in 3 Monaten nicht mehr.